package.json

```diff
@@ -9,13 +9,18 @@
     "cy:run": "cypress run",
     "test:e2e": "npm run cy:run",
     "test:integration": "jest --passWithNoTests",
-    "test": "npm-run-all test:*"
```

Comment:
this should be changed back. we want `npm-run-all test:*` because it runs all of our tests with one command.